frontend/app/views/fragments/event/itemList.scala.html

```diff
@@ -1,6 +1,6 @@
 @(event: model.RichEvent.RichEvent, classes: Seq[String] = Seq.empty)
 
-<a href="@routes.Event.details(event.underlying.ebEvent.slug)"
+<a href="@event.detailsUrl"
```

Comment:
This should have been using the property from the class anyway, rather than distributing the logic to do a slug-based lookup.